tests/ext/sandbox/hook_function/posthook_exit_02.phpt

```diff
@@ -0,0 +1,50 @@
+--TEST--
+DDTrace\hook_function posthook is called at exit (shutdown handler)
```

Comment:
From a white-box perspective it's not any different. From a black-box perspective it's different, though.

Reply:
I guess I'm trying to figure out what specific code path within ddtrace it's testing since the `exit` in the shutdown handler happens after we've already hooked. It seems like it's the same code path as the first exit test, no?

Reply:
It's the same code path, but importantly it's not called twice (two exits in same request).

Reply:
Ah, good call: `ZEND_EXIT` is emitted twice and we should only run the closure for the first one. Thanks for clarifying. :)